src/main/external-resources/renderers/VLC-for-desktop.conf

```diff
@@ -0,0 +1,51 @@
+#----------------------------------------------------------------------------
+# Profile for the VLC desktop application. VLC 2.x has poor DLNA implementation
+# and may not work very well. VLC 3.x is said to improve this a lot, but wasn't
+# yet released when this text was written.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = VLC for desktop
+RendererIcon = VLC.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent=6.2.9200 2/, UPnP/1.0, Portable SDK for UPnP devices/1.6.19 for VLC 32-bit version 2.2.4
+# User-Agent=6.1.7601 2/Service Pack 1, UPnP/1.0, Portable SDK for UPnP devices/1.6.19 for VLC 64-bit version 2.2.4
+# User-Agent=Linux/3.13.0-68-generic, UPnP/1.0, Portable SDK for UPnP devices/1.6.6
+# User-Agent=Darwin/12.5.0, UPnP/1.0, Portable SDK for UPnP devices/1.6.19
+# ============================================================================
+#
+
+UserAgentSearch = Portable SDK for UPnP devices/1\\.\\d{1,2}\\.\\d{1,3}
+
+TranscodeVideo = MPEGTS-H264-AAC
+TranscodeAudio = WAV
+DefaultVBVBufSize = true
+AutoExifRotate = true
+SendFolderThumbnails = false
+SupportedVideoBitDepths = 8,10
+MediaInfo = true
+
+# Supported video formats:
+Supported = f:avi             m:video/avi
+Supported = f:flv             m:video/x-flv
+Supported = f:mkv             m:video/x-matroska
+Supported = f:mov             m:video/quicktime
+Supported = f:mp4             m:video/mp4
+Supported = f:mpegps|mpegts   m:video/mpeg
+Supported = f:rm              m:application/vnd.rn-realmedia-vbr
+Supported = f:wmv             m:video/x-ms-wmv
+
+# Supported audio formats:
+Supported = f:dts    m:audio/vnd.dts
+Supported = f:flac   m:audio/flac
+Supported = f:ogg    m:audio/ogg
+
+# Supported image formats:
+Supported = f:jpg   m:image/jpeg
+Supported = f:png   m:image/png
+Supported = f:gif   m:image/gif
+
+SupportedInternalSubtitlesFormats = ASS,SUBRIP
```

Comment:
@Sami32 AFAIK @valib simply copied and modified the VLC for IOS profile so the subtitles support is an inheritance from that. It might very well be that more types are supported, but I didn't bother to test. I tested external subtitles and found nothing to work with VLC, but I didn't want to take the trouble of muxing different internal subtitle types into files for testing.

It's likely that the support for internal subtitles are the same as when VLC plays a file directly, but there's no way to be sure without testing. The current DLNA implementation in VLC also means it's not that relevant imo, but if you want to test and confirm that's the best solution.